### PR TITLE
The Next Web clone

### PR DESCRIPTION
**Pull request by Jarfsoft**
**HTML elements added:**

- Header
- Body
- Sections
- Footer

**CSS Styles added:**

- Responsive styles to mobile version
- Responsive styles to Ipad version
- Styles to Desktop version

### CODE REVIEW

File: README.md
Comment (teekaytech) [inline README.md]: Preview this page from GitHub and fix the formatting issues. 
- The authors should be separated by space.
- Fix the markdown formatting bugs.

[This](https://guides.github.com/features/mastering-markdown/) might help.

File: README.md
Comment (teekaytech) [inline README.md]: I'm sure this is a copy/paste error because The Next Web has nothing to do with New York Times :smile:. Kindly fix it.

File: assets/styles/style.css
Comment (teekaytech) [inline assets/styles/style.css]: There seems to be a lot of repeated rules here. Always keep your code DRY. Please group like rules together to ensure DRYness in these blocks and all other blocks where such is happening in your code.




[This article](https://speckyboy.com/good-bad-css-practices/) might help in understanding CSS good practices.

File: index.html
Comment (teekaytech) [inline index.html]: The `alt` attribute value is missing. Kindly add it.

**Review by teekaytech — CHANGES_REQUESTED** [review]
# Status: Changes Required :warning: :construction:

## Hi team :wave: 
**This project is looking great :clap:**
- Well composed README file :+1: 
- Descriptive PR title and summary :100: 
- Your linters are green :heavy_check_mark: 
- Nice implementation :tada: 

Overall, this is an awesome job and I'm impressed!. However, we do have some changes to make before the project can be approved (see comments below).

**[ REQUIRED CHANGES ]** 

_I will focus on responsiveness in this review as the main point of this project is to implement a Responsive Web Design._

**Medium version changes (768px - 1023px)** 
- [x] The TNW logo seems to be overlapping on a nav item. Kindly fix this.

Source page:


Your page:


- [x] The `Latest Funding` section aligns 3 articles per row (768px - 1023px).

Source page:


Your page:


**Small version changes (<= 767px)** 

- [x] There seems to be a bug in the footer section. Check the images below:

Source page:


Your page:



* **When you have made the specified changes, please submit another code review.**
* **If you feel like you need help or you have any question(s), I'm @taofeekolalere on slack**

Regards,
[Taofeek Olalere](https://github.com/teekaytech)
Happy Coding :technologist:

**Review by teekaytech — CHANGES_REQUESTED** [review]
## Status: Changes Required :warning: :vertical_traffic_light: 

### Hello team :wave:

__I must say, you've done an amazing job so far. However, there are some changes that are required to be fixed before this project can be approved. Please check my points below and fix them as appropriate.__

**[ REQUIRED CHANGES ]**

- [x] You are still having some repeated rules in your CSS file. Please check my previous comment and refactor as appropriate.



_**Note that this is one of the microverse specific requirements for this project. So it is important you refactor your CSS file.**_




**Please make the required changes and submit another review request when you are done.**

* * *

**_You can reach out to me on twitter (@ola_lere), Github(@teekaytech) or slack (@taofeekolalere )_**
__Happy Coding :smiley:__

File: assets/styles/style.css
Comment (acushlakoncept) [inline assets/styles/style.css]: If you notice there is a lot of repeated `linear-gradient(to top, rgba(0, 0, 0, 1), rgba(0, 0, 0, 0.4))`
One good way to overcome this is to use [CSS Variables](https://www.w3schools.com/css/css3_variables.asp). Example:

```
:root {
  --gradient: linear-gradient(to top, rgba(0, 0, 0, 1), rgba(0, 0, 0, 0.4));
}

/* use it like this */
.hf-img {
  background-image: var(--gradient), url(../images/hf.jpg);
}
```
Now, it becomes very easy to make one change to affect the rest.

**Review by acushlakoncept — CHANGES_REQUESTED** [review]
# Status: Changes Required ♻️

Hello @Jarfsoft and @sumon0002001, Great job! 👏  Both of you have done well on this project.
Let's make a few changes, and we will be done.

- [ ] Fix the HTML validation warnings/errors, use the [w3 Validator](https://validator.w3.org/)

- [ ] Add **.DS_Store** inside your assets folder to **.gitignore** file

- [ ] There is a stranded `head>` tag at the bottom of the page, I'm sure if you fix the HTML validation errors, it should fix it as well. 
fo

- [ ] Like the Previous TSE said, you still have some repeated rules in your CSS file. No worries 😄 I have provided some hints below on your code to help you.

### Cheers and Happy coding! 💻 👏

(Feel free to contact me on slack(@acushla), if you have any additional question)
Uduak Essien
[Twitter](https://twitter.com/acushlakoncept) | [LinkedIn](https://www.linkedin.com/in/acushlakoncept/)

**Review by javitocor — CHANGES_REQUESTED** [review]
# STATUS CHANGES REQUIRED ♻️ 

Good job with the changes, team! 👍  I can see you put a lot of effort into them! 👏 
But we still to implement some changes before the project get approved 💪 

Please, check my comments below 👇  apply the changes and resubmit your project fo a new code review ⏩ 

Good luck with it! 😃 

## Changes

- Screens above 1024px:
   - hard fork, apps, gear.... sections, kindly fix the size of the image and the size of the text in the lastest section
original:
Capture
Your project:
Capture1
- Screens below 1024px:
   - in the latest fundings section the add info, get updates and see more links must be inline
   - Keep working on the images sizes in the hard fork, apps section
- Screen below 768px:
   - in the hard fork, apps... sections, kindly fix the image of the latest section
Capture2
   - Keep working on the images sizes in the hard fork, apps section

**Review by acushlakoncept — CHANGES_REQUESTED** [review]
# Status: Changes Required ♻️

### Great Job, Team!, awesome progress you've made 👏 
Let's make a few changes, and we will be done.

- [ ] You still have some HTML validation warnings/errors to fix, use the [w3 Validator](https://validator.w3.org/)

- [ ] You're yet to "kindly fix the image of the latest section" for screen below 768px as requested by the previous TSE
latest

Apply these changes and resubmit another code review request.

### Cheers and Happy coding! 💻 👏

(Feel free to contact me on slack(@acushla), if you have any additional question)
Uduak Essien
[Twitter](https://twitter.com/acushlakoncept) | [LinkedIn](https://www.linkedin.com/in/acushlakoncept/)

**Review by javitocor — CHANGES_REQUESTED** [review]
# STATUS CHANGES REQUIRED ♻️ 

Hello again, team!
You are making progress! 👏 


But we still need to implement some changes 💪 
Check them below 👇 

## Changes
- You still have some HTML validation warnings/errors to fix, use the [w3 validator](https://validator.w3.org/)
- Screen below 768px, in the hard fork, apps... sections, kindly fix the image of the latest section
- Screens below 1024px, in the latest fundings section the add info, get updates and see more links must be inline
- In the whole page, in hard fork, apps, gear.... sections, kindly fix the size of the image and the size of the text in the lastest section

**Review by teekaytech — COMMENTED** [review]
## Status: Changes Required :warning: :vertical_traffic_light: 

### Hello team :wave:

__I must say, you've done an amazing job so far. However, there are still some minor changes that are required to be fixed before this project can be approved. Please check my points below and fix them as appropriate.__

**[ REQUIRED CHANGES ]**

- [x] In the `Latest Fundings` section, when rendered for medium screen sizes (768px - 1024px), the `Add info`, `Get updates by email` and `See more` items are on the same line, but you have them on separate lines in your clone. Kindly fix it.

Source page:


Your clone:



- [ ] In the latest deals section, the old and new prices are both on the same line. Kindly put them on the same line on your clone.

Source page:


Your page:



**[ OPTIONAL CHANGES ]**

**Please make the required changes and submit another review request when you are done.**

* * *

**_You can reach out to me on twitter (@ola_lere), Github(@teekaytech) or slack (@taofeekolalere )_**
__Happy Coding :smiley:__

**Review by teekaytech — APPROVED** [review]
# Status: Approved :green_circle: 

### Congratulations team, this project has passed all the mandatory requirements. It is therefore approved :heavy_check_mark:

Again, congratulations :tada: You can merge to master.

Do not forget to fill the project completion form.

Good luck with your next task.

**I'm @taofeekolalere on slack.
Happy Coding :man_technologist:**

**Pull request merged**
